oauthenticator/globus.py

```diff
@@ -50,9 +50,8 @@ async def clear_tokens(self, user):
         if state:
             await self.authenticator.revoke_service_tokens(state.get('tokens'))
             self.log.info(
-                'Logout: Revoked tokens for user "{}" services: {}'.format(
-                    user.name, ','.join(state['tokens'].keys())
-                )
+                f'Logout: Revoked tokens for user "{user.name}" services: '
+                ','.join(state['tokens'].keys())
```

Comment:
I don't think this is right:
```py
In [8]: u='u'

In [9]: t=['a','b']

In [10]: (f'{u}: '
    ...: ','.join(t))

Out[10]: 'au: ,b'
```

Reply:
Oh the join attaches to the full string object, not just `','`, nice catch!

Reply:
Reverted in [047c952](https://github.com/jupyterhub/oauthenticator/pull/555/commits/047c952e8f30e2d6e693dea471696b949fe9211c)